Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreRef.cpp

```diff
@@ -782,3 +782,12 @@ void WKWebsiteDataStoreGetAllStorageAccessEntries(WKWebsiteDataStoreRef dataStor
         callback(context, domainArrayRef);
     });
 }
+
+void WKWebsiteDataStoreResetResouceMonitorThrottler(WKWebsiteDataStoreRef dataStoreRef)
```

Comment:
`WKWebsiteDataStoreResetResourceMonitorThrottler`

Reply:
Thanks for finding TYPO!